pkg/data/helpers.go

```diff
@@ -37,3 +45,81 @@ func getFiltersFromStruct(i interface{}, keyName *string) (*expression.KeyCondit
 	}
 	return kb, cb
 }
+
+func putItem(input interface{}, data interface{}, tableName string, expr *expression.Expression) error {
+
+	returnValue := "NONE"
+
+	var putMap *dynamodb.AttributeValue
+
+	var dataModel *model.Account
+	var i *Account
+
+	if tableName == "account" {
+		i, _ = input.(*Account)
+		dataModel, _ = data.(*model.Account)
+
+	}
+	putMap, _ = dynamodbattribute.Marshal(dataModel)
+	_, err := invoke(i.DynamoDB, "PutItem",
```

Comment:
still not sure I get the value of invoke here.

Reply:
I was trying to make this method generic to handle errors. But made it simpler now,